<launch>
  <!-- global PARAMS Change here if needed -->
  <arg name="controller_config" default="$(find schunk_svh_driver)/etc/controller_all_versions.yaml" />

  <!-- Load the User Config file which will overwrite values from default -->
  <rosparam file="$(arg controller_config)" command="load" ns="svh_controller"/>
 
  <!-- Launch actual driver node (if not in simulation)-->
  <node name="svh_controller" pkg="schunk_svh_driver" type="svh_dynamic_parameter_test" output="screen">
  </node>


  
</launch>
